lib/rules/hashbang.js

```diff
@@ -78,7 +78,17 @@ function getShebangInfo(sourceCode) {
     }
 }
 
-/** @type {import('eslint').Rule.RuleModule} */
+/**
+ * @typedef {[
+ *   {
+ *     convertPath?: unknown;
```

Comment:
Could we import this from https://github.com/eslint-community/eslint-plugin-n/blob/master/lib/util/get-convert-path.js#L15

```suggestion
 *     convertPath?: import('../util/get-convert-path').ConvertPath;
```